src/constants.js

```diff
@@ -11,3 +11,4 @@ export const USER_INTERFACE_ID = 'user-interface';
 export const START_QUIZ_BUTTON_ID = 'start-quiz-button';
 export const ANSWERS_LIST_ID = 'answers-list';
 export const NEXT_QUESTION_BUTTON_ID = 'next-question-button';
+export const QUESTION_NUMBER = 3;
```

Comment:
🤔 we don't need a variable for this! Use something inside of `data.js`

Reply:
I created a variable in Data.js